media/js/firefox/features/features-article.es6.js

```diff
@@ -6,13 +6,18 @@
 
 const client = Mozilla.Client;
 const footerCta = document.querySelector('.c-feature-footer');
+const midPageCta = document.querySelector('.c-firefox-midpage-cta');
 
 (function () {
     if (
         (footerCta && client.isFirefoxDesktop) ||
         (footerCta && client.isFirefoxAndroid) ||
-        (footerCta && client.isFirefoxiOS)
+        (footerCta && client.isFirefoxiOS) ||
+        (midPageCta && client.isFirefoxDesktop) ||
+        (midPageCta && client.isFirefoxAndroid) ||
+        (midPageCta && client.isFirefoxiOS)
     ) {
         footerCta.parentNode.removeChild(footerCta);
+        midPageCta.parentNode.removeChild(midPageCta);
```

Comment:
Issue: this throws an error in Firefox when viewing a locale that doesn't feature a mid page CTA:

```
Uncaught TypeError: can't access property "parentNode", midPageCta is null
```

Reply:
I think you can also simplify all this logic to:

```javascript
const client = Mozilla.Client;
const footerCta = document.querySelector('.c-feature-footer');
const midPageCta = document.querySelector('.c-firefox-midpage-cta');

if (midPageCta && client.isFirefox) {
    midPageCta.parentNode.removeChild(midPageCta);
}

if (footerCta && client.isFirefox) {
    footerCta.parentNode.removeChild(footerCta);
}
```